rust-runtime/aws-smithy-http-server/examples/pokemon-service/src/main.rs

```diff
@@ -63,6 +63,9 @@ pub async fn main() {
         .health_check_operation(health_check_operation)
         .build();
 
+    // Unchecked build does not type check whether or not an operation is set but will panic at runtime.
```

Comment:
We can showcase this in the lambda binary better. But for now this new service builder will not be announced, so a Kotlin unit test is better for now.